clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -1793,6 +1793,36 @@ void ItaniumCXXABI::EmitDestructorCall(CodeGenFunction &CGF,
                             ThisTy, VTT, VTTTy, nullptr);
 }
 
+// Check if any non-inline method has the specified attribute.
+template <typename T>
+static bool CXXRecordNonInlineHasAttr(const CXXRecordDecl *RD) {
+  for (const auto *D : RD->noload_decls())
```

Comment:
Please put braces around the `for` loop body. Technically it is a single-statement body but it's long enough that I think braces will be helpful.